WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -436,19 +436,19 @@ extension WordPressAppDelegate {
     private func setupFancyAlertAppearance() {
         let appearance = FancyAlertView.appearance()
 
-        appearance.titleTextColor = WPStyleGuide.darkGrey()
+        appearance.titleTextColor = .neutral(shade: .shade700)
         appearance.titleFont = WPStyleGuide.fontForTextStyle(.title2, fontWeight: .semibold)
 
-        appearance.bodyTextColor = WPStyleGuide.darkGrey()
+        appearance.bodyTextColor = .neutral(shade: .shade700)
         appearance.bodyFont = WPStyleGuide.fontForTextStyle(.body)
 
         appearance.actionFont = WPStyleGuide.fontForTextStyle(.headline)
         appearance.infoFont = WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .semibold)
         appearance.infoTintColor = WPStyleGuide.wordPressBlue()
```

Comment:
The app delegate still has quite a few references to old colors, for example here. Do a search for `WPStyleGuide.` in here (I know some are in feature-flagged non-Muriel code, but some aren't).